### DISCUSSION TITLE
FLSUN SR Settings for OctoPrint

### DISCUSSION THREAD

**Discussion by dominickp** [blackfyre/flsun.community#13] [Q&A]
I am very new to 3D printing (the SR is my first printer). I have been using Cura 4.4 that came with the printer and I've been moving prints with the SD card. I tried to setup OctoPrint today and I tried to use the settings in the Cura profile to setup the profile in OctoPrint.

I printed a test print and almost immediately, the printer arranged it's arms in a way that it couldn't handle and two of the arms snapped off of their balls. Luckily nothing broke and the printer seems to be working from what I can tell.

But I'm really confused, I don't understand what it going on. If I plug in OctoPrint and connect it, the axis are not moving correctly. If I move it down, it goes off to one side (2 of the axis go down while one just stays there). If I restart the printer but leave OctoPrint unplugged, it behaves normally.

Is there a resource where I should look to see how to setup a SR profile?

Here are my current settings in OctoPrint:

![228640755_4116021358475135_7924453235523920121_n](https://user-images.githubusercontent.com/4555880/127753801-b8c32aec-7dea-49fa-b4a3-624d5047140b.jpg)
![228704755_4116021368475134_8300527044447079008_n](https://user-images.githubusercontent.com/4555880/127753802-1eeac9c7-66da-406a-b6c8-2328bef2ca29.jpg)
![229455198_4116021315141806_7341015693350373186_n](https://user-images.githubusercontent.com/4555880/127753803-397887a9-2ee3-4911-b0af-7c1a5d02ed1e.jpg)

**Comment by blackfyre** [comment]
Actually, it wasn't octoprint that messed up, but the slicer. Unless you've sliced with octoprint (which is being phased out last I've checked)...
Octoprint only streams the gcode, the SD card is only a different source in this sense.
